app/controllers/alerts_list_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 # Controller for showing Alert records
 class AlertsListController < ApplicationController
+  include AlertsHelper
```

Comment:
So do we have a consensus whether or not the module should be explicitly included in the controller?